src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/RespondentTellSomethingElseService.java

```diff
@@ -200,58 +193,39 @@ public void sendClaimantEmail(CaseDetails caseDetails) {
         boolean isWelsh = featureToggleService.isWelshEnabled()
```

Comment:
claimantPersonalisation() does a look up of this value again. Can you perhaps send this value into it as a parameter as well?